appveyor.yml

```diff
@@ -4,9 +4,18 @@ clone_depth: 50
 os: Visual Studio 2015
 
 environment:
+  PYTEST_DIRECTIVES: --skip-slow
```

Comment:
question: to disable skip-slow temporarily e.g. in a PR, can we just delete `--skip-slow` or does there need to be something there so it's not empty